meta/meta.go

```diff
@@ -101,6 +101,25 @@ const (
 	MaxGlobalID = MaxInt48 - 1000
 )
 
+// DDLTableVersion indicates the DDL table version type.
+type DDLTableVersion int
+
+const (
+	// InitDDLTableVersion is the original version.
+	InitDDLTableVersion DDLTableVersion = 0
+	// BaseDDLTableVersion is for support concurrent DDL, it added tidb_ddl_job, tidb_ddl_reorg and tidb_ddl_history.
+	BaseDDLTableVersion DDLTableVersion = 1
+	// MDLTableVersion is for support MDL tables.
+	MDLTableVersion DDLTableVersion = 2
+	// BackfillTableVersion is for support distributed reorg stage, it added tidb_ddl_backfill, tidb_ddl_backfill_history.
+	BackfillTableVersion DDLTableVersion = 3
```

Comment:
an optional suggestion, we could insert current ddlTableVersion into mysql.tidb table, just like `tidb_server_version`

Reply:
This is supposed to use SQL, there's no session here

Reply:
we coudl do it in `doDMLWorks()` func, get the current DDLTableVersion and insert it into `mysql.tidb` table.